candi/cloud-providers/aws/openapi/cluster_configuration.yaml

```diff
@@ -71,6 +74,18 @@ apiVersions:
                 description: Instance disk size in gigabytes.
                 example: 20
                 type: integer
+              etcdDisk:
+                type: object
+                default: {"sizeGb": 150, type: "gp2"}
```

Comment:
why we need this default if we have defalts for sizeGb and type parameters ?

Reply:
Left defaults only here. The defaults for etcdDisk.sizeGb and etcdDisk.type do not work.